tests/setup_suite.bash

```diff
@@ -3,7 +3,7 @@
 set -eu
 
 function setup_suite {
-  export BATS_TEST_TIMEOUT=120
+  export BATS_TEST_TIMEOUT=300
```

Comment:
Did we encounter a test case which exceeds this? I think we'd like to identify the cause behind such failures, or atleast verify that that delay is expected. I'd prefer to keep this 120 for now and bump it when we identify causes or expectation.

Reply:
I see. 

So the 300 second wait is anyways a no-op because bats would cancel at the 120 second mark. I think the 300 may have been set arbitrarily, given the default is 30 second and we wanted something more.

We can bump down the kubectl wait to 120. Bump up the Bats test timeout to 150.